docs/guide/advanced/post_bloom.md

```diff
@@ -25,13 +29,14 @@ Engine3D.startRender(renderJob);
 | --- | --- | --- |
 | blurX | number | The horizontal blur radius of the screen.|
 | blurY | number |  The vertical blur radius of the screen.|
-| intensity | number |  	The intensity of the effect.|
-| brightness | number |  The brightness of the effect.|
+| strength | number |  Omni intensity.|
```

Comment:
this chapter is about bloom. thus it can use "Bloom intensity","Bloom radius" ,"Bloom intensity filtering threshold"